Create trim.md (#1914)
Codecademy/docs: content/php/concepts/string-functions/terms/trim/trim.md

---
Title: 'trim()' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed

> Description: 'Strip whitespace (or other characters) from the beginning and end of a string' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)

Reviewer: ```suggestion
Description: 'Strips whitespace (or other characters) from the beginning and end of a string.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```

Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
  - 'String'
  - 'Functions'
CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
  - 'learn-php'
  - 'path/computer-science'
---

> `trim()` returns a string with whitespace stripped from the beginning and end of string.

Reviewer: ```suggestion
The **`trim()`** function returns a string with whitespace stripped from the beginning and end of string.
```

> ## Pseudo Code:

Reviewer: ```suggestion
## Syntax
```

> `trim(string $string, string $characters = " \n\r\t\v\x00"): string`

Reviewer: This should be in a ` ```pseudo ` code block, not single backticks.

> Parameters:

Reviewer: Rewrite this Parameters section by removing the line of dashes and organizing the parameters in a bulleted list like this:

- `string`: The string that will be trimmed.
- `characters`: Optionally, the stripped characters can also be specified using the characters parameter.

Also, rewrite to remove the use of the "you" pronoun.

------------------------------------------------------
string
The string that will be trimmed.

characters
Optionally, the stripped characters can also be specified using the characters parameter. Simply list all characters that you want to be stripped. With .. you can specify a range of characters.

> ## Syntax

Reviewer: Remove this section.

`$trimmed = trim($text, " \t.");`

> ## Codebyte Example (if applicable)

Reviewer: ```suggestion
## Codebyte Example
```

> ```

Reviewer: This block needs to start with ` ```codebyte/php `.
It also needs a sentence preceding it, describing what it does.

---
Title: 'trim()'
Description: 'Strips whitespace and/or other characters from the beginning and end of a string.'
Subjects:
  - 'Computer Science'
  - 'Web Design'
  - 'Web Development'
Tags:
  - 'String'
  - 'Functions'
CatalogContent:
  - 'learn-php'
  - 'path/computer-science'
---

The **`trim()`** function returns a string with whitespace stripped from the beginning and end of the string.

## Syntax

```pseudo
trim($string, $characters)
```

Parameters:

- `$string`: The string that will be trimmed.
- `$characters`: Optionally, the stripped characters can be specified using the characters parameter. A range of characters can be specified with `..`. By default, it will strip:

>   - Spaces `" "` \n\r\t\v\x00"  

Reviewer 2: ```suggestion
  - Spaces `" "`
```
